riptide-spring-boot-autoconfigure/README.md

```diff
@@ -300,9 +311,15 @@ riptide:
         tags:
           peer.service: example
         propagate-flow-id: true
+      telemetry:
+        enabled: true
+        tags:
```

Comment:
To be consistent with the standard, this should be named `attributes` instead of `tags`

Reply:
`attributes` is what was named in the `RiptideProperties.Telemetry` class. Guess this was just a typo.

```suggestion
        attributes:
```